raml/api.raml

```diff
@@ -0,0 +1,15 @@
+#%RAML 1.0
+---
+title: RAML API
+baseUrl: /
+mediaType: application/json
+
+/api:
```

Comment:
@deepakgc The mock is now accessible via `/api` path. Does it make sense to make it accessible via `/` (at least in this demo) because it is the entrypoint whenever a new service is executed (for people who barely read the documentation)?

Reply:
Agree! done